### ISSUE TITLE
Compiling SCSS to CSS issues

### ISSUE THREAD

**Issue by apcrane** [zurb/foundation-sites-6#273]
Hi, I've been trying to set up a test project compiling the SCSS using various methods and all seem to be having trouble with more than two level deep @import rules. So far I have tried running Koala App (uses SASS version 3.4.19), Compass App (SASS 3.4.13), PrePros (SASS version 3.4.13) and Compass from the command line (SASS version (3.4.19). All are compiling some of the CSS, but only a small amount (see attached text file).

As an example the foundation.scss file contains @import 'grid/grid', however none of the CSS from the secondary imports is compiled i.e. @import 'row';. But if I add a test class into this _grid.scss file, then this is compiled. 

[foundation.txt](https://github.com/zurb/foundation-sites-6/files/31707/foundation.txt)

I have also tried using ScoutApp (which I have used for all other foundation projects up to v5), but this the SASS version (3.2.1) is out of date and won't compile the SCSS, so this is probably a no go until this is updated, error example:

Invalid CSS after " top": expected ")", was ": nth($val, 1),")

My preference is to use a GUI such as Koala (http://koala-app.com/) or Compass App (http://compass.kkbox.com/) to compile this code since the designers here won't be comfortable with the command line interface such as gulp or compass.

I am running Windows 10 and will be using the compiled CSS in a .NET MVC project.

I look forward to hearing back from you about this.

**Comment by apcrane** [comment]
I have now managed to get the SCSS compiling all CSS with with compass with one config.rb file :) I also did manage to get this working with gulp, although this seemed to have so many dependencies.

Although would be nice if a GUI worked, but I appreciate this is up to the owners of the products I've been trying.

**Comment by apcrane** [comment]
The compass.app GUI is now working correctly (http://compass.kkbox.com/). Not entirely sure why this wasn't working earlier but could have been a folder permissions issue and I also did do a pull down of the latest foundation6 version. 

Worth noting that for anyone else trying to compile the scss neither Koala.app or Scout.app work correctly. Koala.app doesn't recognize multiple levels of @import rules and Scout.app is too older version of SASS to work with Foundation6.

So in conclusion, if you are looking to basically compile the scss into css using Windows then stick to Compass via the Command Window or use the command app.

**Comment by gakimball** [comment]
The `@import` statements on their own don't export any CSS. There's a series of mixins, one for each component, that print out the default CSS.

Here's what a full `app.scss` would look like:

```scss
@import 'foundation';

@include foundation-global-styles;
@include foundation-grid;
@include foundation-typography;
@include foundation-button;
@include foundation-forms;
@include foundation-visibility-classes;
@include foundation-float-classes;
@include foundation-accordion;
@include foundation-badge;
@include foundation-breadcrumbs;
@include foundation-button-group;
@include foundation-callout;
@include foundation-close-button;
@include foundation-drilldown-menu;
@include foundation-dropdown;
@include foundation-dropdown-menu;
@include foundation-flex-video;
@include foundation-label;
@include foundation-media-object;
@include foundation-menu;
@include foundation-off-canvas;
@include foundation-orbit;
@include foundation-pagination;
@include foundation-progress-bar;
@include foundation-slider;
@include foundation-sticky;
@include foundation-reveal;
@include foundation-switch;
@include foundation-table;
@include foundation-tabs;
@include foundation-thumbnail;
@include foundation-title-bar;
@include foundation-tooltip;
@include foundation-top-bar;
```

**Comment by apcrane** [comment]
OK thanks for confirming that to me.
